Enterprise-CMCS/macpro-mdct-mfp
# Remove br tags from review and submit page; cleanup styling

### Description

- Remove `<br>` tags from additionalInfo section for WP and SAR submit pages
- Change SAR verbiage list type
- Align list styling
  - The lists for WP and SAR were different, and are different in FIGMA as well
  - I tightened them to what I thought looked good, and more aligned with SAR styling
    - If you think I overstepped let me know!

WP Before:
<img width="1505" alt="Screenshot 2024-05-30 at 11 10 51 AM" src="https://github.com/Enterprise-CMCS/macpro-mdct-mfp/assets/57802560/cfa8b8ea-9f5d-408a-9ec3-6ab6b12e1e73">

WP After:
<img width="1502" alt="Screenshot 2024-05-30 at 11 10 40 AM" src="https://github.com/Enterprise-CMCS/macpro-mdct-mfp/assets/57802560/78595f79-e499-4894-8c73-f9721132a236">

SAR Before:
<img width="1509" alt="Screenshot 2024-05-30 at 11 08 41 AM" src="https://github.com/Enterprise-CMCS/macpro-mdct-mfp/assets/57802560/6f57fed8-ee79-4775-bfe4-53cddbd0be41">

SAR After:
<img width="1508" alt="Screenshot 2024-05-30 at 11 08 59 AM" src="https://github.com/Enterprise-CMCS/macpro-mdct-mfp/assets/57802560/c628f7be-e54a-4b7c-928d-86624e2f4ba9">

### Related ticket(s)


CMDCT-3609

---
### How to test

https://d34ws8l2dczce1.cloudfront.net/
- Log in as `stateuser@test.com`
- View the review & submit page of the WP and the SAR (there is already a submitted version of each)
- Verify there are no break tags in the "What happens now?" section and below

---
### Pre-review checklist
